src/App.tsx

```diff
@@ -1,59 +1,11 @@
 import './App.css';
 
-import { ReactElement, useState } from 'react';
-import { useQuery } from 'react-query';
+import { ReactElement } from 'react';
```

Comment:
App doesn't need the `ReactElement` to be used to type it. It will work fine without it like `function App() { ... }`